apps/prairielearn/src/lib/question-render.ts

```diff
@@ -347,7 +347,7 @@ function buildLocals({
         locals.showTrueAnswer = true;
       }
     }
-    if (!assessment.allow_real_time_grading) {
+    if (assessment_question?.allow_real_time_grading === false) {
```

Comment:
There is never a scenario in which `assessment` is defined but `assessment_question` is not, right? It would be great to encode that into the type system/a runtime check, as that is confusing to me.

Reply:
I'm reasonably confident that #12742 will catch this; can we wait and see?